src/dotvvm-vscode/CHANGELOG.md

```diff
@@ -1,9 +1,5 @@
-# Change Log
+# Changes
 
-All notable changes to the "dotvvm-vscode" extension will be documented in this file.
-
-Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
-
-## [Unreleased]
-
-- Initial release
\ No newline at end of file
+## 0.9.0 (first version after a major rewrite)
+* Added DotVVM language server
+* Added Intellisense and auto-completion for controls and properties
```

Comment:
```suggestion
* Added auto-completion for controls and properties
* Updated dothtml grammar for DotVVM 4.0
* Added expand selection support
```

Intellisense is Visual Studio not Code word for autocomplete :D

Reply:
I am alright with the change, though even Microsoft seem to use it for VS Code as well: https://code.visualstudio.com/docs/editor/intellisense